tree/treeplayer/src/TFileDrawMap.cxx

```diff
@@ -577,7 +577,10 @@ void TFileDrawMap::PaintDir(TDirectory *dir, const char *keys)
          while ((leaf = (TLeaf*)nextb())) {
             TBranch *branch = leaf->GetBranch();
             color = branch->GetFillColor();
-            if (color == 0) color = 1;
+            if (color == 0) {
+               gPad->IncrementPaletteColor(1, "pfc");
```

Comment:
Should we reset the palette increment at the end?

Reply:
Indeed not, this is already handled by TPad.